config/schedule.rb

```diff
@@ -0,0 +1,20 @@
+# Use this file to easily define all of your cron jobs.
+#
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+# Learn more: http://github.com/javan/whenever
+
+set :output, "/home/rails/waste-exemptions-back-office/shared/log/whenever_cron.log"
+set :job_template, "/bin/bash -l -c 'eval \"$(rbenv init -)\" && :job'"
+
+# Only one of the AWS app servers has a role of "db"
+# see https://gitlab.envage.co.uk/dst-south/waste-exemptions-deployment/blob/master/config/deploy.rb#L12
```

Comment:
Also meant to catch that this line needs updating
```suggestion
# see https://gitlab-dev.aws-int.defra.cloud/open/rails-deployment/blob/master/config/deploy.rb#L69
```

Reply:
👍